metacat-metadata-mysql/src/test/groovy/com/netflix/metacat/metadata/mysql/MySqlLookupServiceSpec.groovy

```diff
@@ -0,0 +1,79 @@
+/*
+ *       Copyright 2017 Netflix, Inc.
+ *          Licensed under the Apache License, Version 2.0 (the "License");
+ *          you may not use this file except in compliance with the License.
+ *          You may obtain a copy of the License at
+ *              http://www.apache.org/licenses/LICENSE-2.0
+ *          Unless required by applicable law or agreed to in writing, software
+ *          distributed under the License is distributed on an "AS IS" BASIS,
+ *          WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *          See the License for the specific language governing permissions and
+ *          limitations under the License.
+ */
+
+package com.netflix.metacat.metadata.mysql
+
+import com.google.common.collect.Lists
+import com.netflix.metacat.common.QualifiedName
+import com.netflix.metacat.common.server.model.Lookup
+import com.netflix.metacat.common.server.properties.Config
+import com.netflix.metacat.testdata.provider.DataDtoProvider
+import org.springframework.jdbc.core.JdbcTemplate
+import spock.lang.Ignore
+import spock.lang.Specification
+import spock.lang.Unroll
+
+import java.time.Instant
+
+/**
+ * Tests for MysqlUserMetadataService.
+ * TODO: Need to move this to integration-test
+ * @author amajumdar
+ */
+
+class MySqlLookupServiceSpec extends Specification {
+    MySqlLookupService mySqlLookupService
+
+    Config config
+    JdbcTemplate jdbcTemplate
+
+    def setup() {
+        config = Mock(Config)
+        jdbcTemplate = Mock(JdbcTemplate)
+
+        mySqlLookupService = new MySqlLookupService(config, jdbcTemplate)
+    }
+
+    @Unroll
+    def "test includeValues"() {
```

Comment:
Can you add some additional tests featuring duplicates and multiple call iterations

Reply:
Sorry, could you elaborate more?

I think I have covered them also in the functional test.

Reply:
ok looks like the functional test can cover the iterations part, could you add another unit test featuring if a caller tries to add duplicate tag in one call like pass in the same tag multiple times in the input list